### PR TITLE
nixos/virtualbox-image: increase disk to 50G

### PR DESCRIPTION
**Pull request by c0bw3b**
###### Motivation for this change

Follow-up to #46649 and #48151
The official demo VM needs to offer more space for practical usage.

This PR extends the max possible disk size from 10GB to 50GB.
The generated OVA size does not change and since the VM has dynamic provisioning it still takes the same size (around 4GB) the first time the VM is imported and launched.

100GB breaks cptofs but 50GB is fine and benchmarks shows it takes the same time as building the demo VBox VM with a 10GB disk. See below.

Also done:
+ enabled VM sound output (was muted by default)
+ set USB controller in USB2.0 mode (was in default USB1.1)
+ add manifest file in the OVA as it allows integrity checking on imports

/cc @brainrape @bjornfor @matthewbauer 

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

I've run many OVA builds over the last days using `time nix-build -I nixpkgs=. -A ova '<nixpkgs/nixos/release.nix>' --option max-silent-time 7200` for some basic benchmarking.
TL;DR; --> with 10GB / 25GB / 50GB all builds took around 16minutes on my machine.

I did one try with 100GB and I had to abort the build after more than one hour... `cptofs` was taking 100% of one core but was not going anywhere... I'm not able to troubleshoot cptofs so let's stick to 50GB for now. :smirk_cat: 

---

### CODE REVIEW

**Review by matthewbauer — APPROVED** [review]

**Comment by c0bw3b** [comment]
Thanks.
(it's also sort of a worklog for myself if I ever want to retry something similar :p )

**Comment by c0bw3b** [comment]
:tada: first OVA with some real estate [available on Hydra](https://hydra.nixos.org/build/83289982)
